ocpmodels/datasets/co_cu_md.py

```diff
@@ -160,6 +160,36 @@ def get_dataloaders(self, batch_size=None, shuffle=True):
 
         return train_loader, val_loader, test_loader
 
+    def ase_atoms_to_batch(self, atoms):
+        generator = AtomicFeatureGenerator(None).extract_atom_features(atoms)
+        embedding, distance, index, positions, atomic_numbers = generator
+        edge_index = [[], []]
+        edge_attr = torch.FloatTensor(
+            index.shape[0] * index.shape[1], distance.shape[-1]
+        )
+        for j in range(index.shape[0]):
+            for k in range(index.shape[1]):
+                edge_index[0].append(j)
+                edge_index[1].append(index[j, k])
+                edge_attr[j * index.shape[1] + k] = distance[j, k].clone()
+        edge_index = torch.LongTensor(edge_index)
+        data = Batch.from_data_list(
+            [
+                Data(
+                    x=embedding,
+                    edge_index=edge_index,
+                    edge_attr=edge_attr,
+                    pos=positions,
+                    atomic_numbers=atomic_numbers,
+                    natoms=torch.tensor([positions.shape[0]]),
+                    y=torch.tensor([0]),
+                    force=torch.tensor([0]),
```

Comment:
We do this here and in the dataset's `process` method. Does the job in bypassing the need for an energy/force...not sue if we would need to change this in the future for clarity?

Reply:
Removed `y` and `force` altogether from this function for now and made appropriate changes elsewhere to make sure that works fine for evaluation. I think it should be clearer now.